### DISCUSSION TITLE
Performance regarding assigment 1

### DISCUSSION THREAD

**Discussion by sorobedio** [seominjoon/kaist-ai605#74] [Q&A] [answered]
1-Does higher performance matter?
2-Should we evaluate the model only on the validation dataset and not the test dataset?
3-Is it okay if the baseline model outperforms the others models?

**Comment by seominjoon** [comment, marked as answer]
1. We expect that the performance is within a reasonable range, but you don't have to worry about it too much.
2. Yes, please use the validation dataset. You are welcome to include test set result as well, though.
3. Yes, that's fine.

**Reply by sorobedio** [reply]
ok thank you

**Reply by emjay73** [reply]
Can I get a reference for a "reasonable range"?
I wonder I've done my homework correctly and by referring to the value I want to check whether I'm on the right track.

**Reply by seominjoon** [reply]
I will resolve this issue now that the solution is up. I tried to avoid telling you reasonable accuracy, but perhaps this is not a good idea. I will try to give you more hints in the next two assignments.
